.github/workflows/release.yml

```diff
@@ -17,27 +17,22 @@ jobs:
     name: Deploy release
     runs-on: ubuntu-latest
     needs: [ci, docs]
-    env:
-      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      # Needs to be an admin token to get around branch protection.
-      GH_TOKEN: ${{ secrets.GH_TOKEN_REDALLEN }}
-      GH_PR_TOKEN: ${{ secrets.GH_PR_TOKEN }}
     steps:
       - name: Check out project
         uses: actions/checkout@v4
         with:
+          # Fetch all history for all branches and tags, which is needed for the release script.
           fetch-depth: 0
-          # Needs to be an admin token to get around branch protection.
+          # Pass in an administrator token to get around branch protection.
           token: ${{ secrets.GH_TOKEN_REDALLEN }}
 
-      # Fetches all tags from 'origin' and updates local tags, only fetches the latest commit.
-      - name: Fetch tags
-        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
-
```

Comment:
I've remove this as `fetch-depth: 0` already fetches all the tags, see https://github.com/actions/checkout/issues/217#issuecomment-900923602.

Reply:
Fetch depth 0 was added (i believe) to make sure we weren't republishing EVERY package in the monorepo each time a single change went into one package. It only republished the relevant packages. I guess we can watch to make sure this doesnt undo that effort now that it's removed.

Reply:
So from what I understand Lerna should take care of this for us, but I can see this issue happening if Lerna does not detect any tags. That perhaps happened before if the `fetch-depth` wasn't set in the action, in which case the default is `1`, which would only fetch the latest commit, and perhaps not all the tags.

The [Lerna docs](https://lerna.js.org/docs/features/distribute-tasks#cicd-examples) link to [an Nx example](https://nx.dev/ci/recipes/set-up/monorepo-ci-github-actions) that seem to operate on the same premise. From what I can gather from community blog posts it seems that `fetch-depth: 1` should be enough for Lerna to do the right thing.

Reply:
Since fetch-depth has 1 as a default value, this should be okay. let's just keep an eye on it.